src/yb/tools/ts-cli.cc

```diff
@@ -658,11 +658,13 @@ static int TsCliMain(int argc, char** argv) {
     string tablet_id = argv[2];
     RETURN_NOT_OK_PREPEND_FROM_MAIN(client.FlushTablets(tablet_id, true /* is_compaction */),
                                     "Unable to compact tablet");
+    std::cout << "Tablet " << tablet_id << " successfully compacted";
```

Comment:
Don't forget to also add `<< std::endl;` at the end for these

Reply:
Hi @hulien22 ,
I've added all your suggested changes, can you review it again? thanks.